app/models/journaled/client.rb

```diff
@@ -0,0 +1,56 @@
+class Journaled::KinesisClient
```

Comment:
was this file supposed to go away? looks like it's all in kinesis_client.rb now

Reply:
Yup! Weird copy paste artifact I think, nice catch!